### PR TITLE
Attempt at new Router startup message

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: apollo-router/src/executable.rs [hunk 1]
```diff
@@ -217,32 +217,34 @@ pub async fn rt_main() -> Result<()> {
         _ => {
             return Err(anyhow!(
                 r#"
-    💫 Apollo Router requires a supergraph to be set using '--supergraph':
+===
+APOLLO ROUTER v{whatever}
```
Comment (StephenBarlow) [inline apollo-router/src/executable.rs:221]: Ideally this version number could pop in dynamically?

**Comment by StephenBarlow** [comment]
This mostly cleans up and shortens the crux of what's already there, but maybe we want to introduce some other types of content, like a table of supported options a la Rover? Rover in general has a very nice splash output and even gets some colors in there, I think via whatever CLI library it uses under the hood...

Anyway what I'm saying is, happy for us to explore more transformative transformations here.

**Comment by BrynCooke** [comment]
We could allow ` --supergraph` flag to also take a URL?

**Comment by bnjjj** [comment]
@BrynCooke It was already suggested by @o0Ignition0o https://github.com/apollographql/router/pull/377

File: apollo-router/src/executable.rs (hunk 1 above)
Comment (BrynCooke) [reply apollo-router/src/executable.rs:221]: Also related to #793

**Review by bnjjj — APPROVED** [review]

**Pull request merged**